### PR TITLE
Remove the HTTP endpoint for Notary Signer

### PR DESCRIPTION
**Pull request by cyli**
Since its' not used by anything.  The signer is generally kept hidden in notary service deployments anyway, and the GRPC interface is the main supported interface.
cc @NathanMcCauley 

I also wanted to check if anyone was doing anything with the HTTP interface first, though.  @andrewhsu @ecordell @HuKeping @mtrmac?

### CODE REVIEW

**Comment by HuKeping** [comment]
👍

File: docs/reference/signer-config.md [hunk 1]
```diff
@@ -72,22 +70,6 @@ Example:
...
-			traffic on.  Examples:
-			<ul>
-			<li><code>":4444"</code> means listen on port 4444 on all IPs (and
-				hence all interfaces, such as those listed when you run
-				<code>ifconfig</code>)</li>
-			<li><code>"127.0.0.1:4444"</code> means listen on port 4444 on
-				localhost only.  That means that the server will not be
-				accessible except locally (via SSH tunnel, or just on a local
-				terminal)</li>
-			</ul>
```
Comment (riyazdf) [inline docs/reference/signer-config.md]: thank you for updating docs too! 👍

**Comment by riyazdf** [comment]
this is awesome, LGTM

**Comment by endophage** [comment]
All the LGTMs!

**Comment by SvenDowideit** [comment]
@endophage is this for v0.4.0?

**Comment by endophage** [comment]
@SvenDowideit given it has been merged, yes it will be included in 0.4.0. Note it's only the *signer* that is losing plain HTTP endpoints, and nobody but the server was talking to the signer (and the server was only using GRPC).

**Comment by SvenDowideit** [comment]
cool - the docs tooling uses the milestone to determine if a PR should be published, so I've added 0.4

**Pull request merged**